test/CommandLine.Test/PushCommandTest.cs

```diff
@@ -38,7 +38,7 @@ private static IPackageSourceProvider CreateSourceProvider()
         }
 
         [Theory]
-        [InlineData(new object[] { @"X:\test\foobar.symbols.nupkg", @"http://nuget.gw.symbolsource.org/Public/NuGet" })]
+        [InlineData(new object[] { @"X:\test\foobar.symbols.nupkg", @"https://nuget.smbsrc.net/" })]

```

Comment:
removed all occurrences of `foo`, `bar` and `foobar` in PushCommandTest.cs in https://github.com/NuGet/NuGet2/commit/fa862eff17059b02e48cab7845b0d94101263192